packages/pwa-kit-dev/src/configs/jest/jest.config.js

```diff
@@ -35,6 +35,6 @@ module.exports = {
         }
     },
     transform: {
-        '^.+\\.(js|jsx|ts|tsx)$': path.join(__dirname, 'jest-babel-transform.js')
+        '^.+\\.(js|jsx|ts|tsx|mjs)$': path.join(__dirname, 'jest-babel-transform.js')
```

Comment:
There is a dependency that has mjs from ark-ui. we want to be able to transform it